dev/tasks/tasks.yml

```diff
@@ -969,6 +969,8 @@ tasks:
   r-binary-packages:
     ci: github
     template: r/github.packages.yml
+    params:
+      custom_version: Unset
```

Comment:
This is a minor fix for an issue I noticed in this PR, namely that for params to work they have to be listed in `tasks.yml`. This also necessitates the `replace(` thing in the templates themselves as even empty params in `task.yml` are `None` and `default` does not trigger on `None`.